openmetadata-ui/src/main/resources/ui/src/components/Ingestion/IngestionRecentRun/IngestionRecentRuns.component.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Popover, Skeleton, Space } from 'antd';
+import { capitalize } from 'lodash';
+import React, {
+  FunctionComponent,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { useTranslation } from 'react-i18next';
+import { getRunHistoryForPipeline } from '../../../axiosAPIs/ingestionPipelineAPI';
+import {
+  IngestionPipeline,
+  PipelineStatus,
+} from '../../../generated/entity/services/ingestionPipelines/ingestionPipeline';
+import {
+  getCurrentDateTimeMillis,
+  getDateTimeFromMilliSeconds,
+  getPastDaysDateTimeMillis,
+} from '../../../utils/TimeUtils';
+
+interface Props {
+  ingestion: IngestionPipeline;
+  classNames?: string;
+}
+const queryParams = {
+  startTs: getPastDaysDateTimeMillis(1),
+  endTs: getCurrentDateTimeMillis(),
+};
```

Comment:
Move this to constant.

Reply:
It will be tricky as it will not share updated timestamp at when this component rendered